core/src/main/java/org/axonframework/queryhandling/SimpleQueryBus.java

```diff
@@ -153,6 +153,12 @@ private boolean unsubscribe(String queryName,
                     invocationSuccess = true;
                 } catch (NoHandlerForQueryException e) {
                     // Ignore this Query Handler, as we may have another one which is suitable
+                } catch (Throwable e) {
```

Comment:
My hunch is that we want to make this `Exception` i.o. `Throwable`, as we're catching `Errors` this way too for local usage, which the `SimpleQueryBus` is meant for.

Reply:
For Command replies we are currently catching `Exception`, not `Throwable`, as far as I am aware, because of the aforementioned `Error`. Hence why I was pointing this out.
Was discussing/thinking-out-loud-on this with @abuijze couple of hours ago, I don't feel we have come to a conclusion.

In interest if time, I think we should go for a merge at this point to have an RC ready, and discuss this tomorrow during our daily.